README.md

```diff
@@ -28,6 +28,8 @@ To further configure your plugin, use the following custom parameters in your `s
 | `site`                 | Set which Datadog site to send data, this is only used when `flushMetricsToLogs` is `false` or `addExtension` is `true`. Possible values are `datadoghq.com`, `datadoghq.eu`, `us3.datadoghq.com` and `ddog-gov.com`. The default is `datadoghq.com`.                                                                                                                                                               |
 | `apiKey`               | Datadog API Key, only needed when `flushMetricsToLogs` is `false` or `addExtension` is `true`. For more information about getting a Datadog API key, see the [API key documentation][3].                                                                                                                                                                                                                            |
```

Comment:
I would clarify that this API key (the API KMS key option as well) adds the Datadog API key directly to the user's Lambda functions as an environment variable.